package.json

```diff
@@ -1,8 +1,12 @@
 {
   "name": "js-mainframe",
+  "productName": "MainframeOS",
```

Comment:
Could you revert these changes please? We shouldn't add `electron` as a dependency and build setup here, better use the launcher package for this.

Reply:
reverted, remnants of exploring, sorry